Source/Scene/CreditDisplay.js

```diff
@@ -424,7 +385,15 @@ define([
         Check.defined('credit', credit);
         //>>includeEnd('debug');
 
-        if (!credit.showOnScreen) {
+        if (credit.text === 'Cesium ion') {
+            // We don't want to clutter the screen with the Cesium logo and the Cesium ion
+            // logo at the same time. Since the ion logo is required, we just replace the
+            // Cesium logo.  This will also add the logo if the Cesium one was removed.
+            this._currentFrameCredits.imageCredits[cesiumCredit.id] = credit;
+            if (defined(cesiumCredit.parentNode)) {
```

Comment:
this doesn't look right.  Should this be `defined(cesiumCredit.element.parentNode)`?

Reply:
I'm pretty sure this isn't needed and was just left over from an earlier iteration of me trying to fix an issue with credit handling.  I took it out and everything appears to work fine.  I'm going to just remove it